aws-athena/views/reporting-vw_ratio_stats.sql

```diff
@@ -38,18 +38,45 @@ town_names AS (
     FROM {{ source('spatial', 'township') }}
 ),
 
+final_model_parsed AS (
+    SELECT
+        fm.year,
+        fm.run_id,
+        fm.is_final,
+        CASE fm.triad_name
+            WHEN 'City' THEN '1'
+            WHEN 'North' THEN '2'
+            WHEN 'South' THEN '3'
+        END AS triad_code,
+        CAST(
+            JSON_PARSE(fm.township_code_coverage) AS ARRAY<VARCHAR>
+        ) AS townships
+    FROM {{ ref('model.final_model') }} AS fm
+),
+
 model_values AS (
     SELECT
         ap.meta_pin AS parid,
-        CAST(CAST(ap.meta_year AS INT) + 1 AS VARCHAR) AS year,
+        ap.year,
```

Comment:
`meta_year` is the year of characteristics, `year` is the actual year of modeling/values.